--- gcc-4.9.0.orig/configure	2014-03-26 04:19:44.000000000 -0500
+++ gcc-4.9.0-1fixlibdirfftarget/configure	2014-05-21 10:10:01.906517000 -0500
@@ -2818,12 +2818,7 @@
 appdirs=""
 
 # Define is_cross_compiler to save on calls to 'test'.
-is_cross_compiler=
-if test x"${host}" = x"${target}" ; then
-  is_cross_compiler=no
-else
-  is_cross_compiler=yes
-fi
+is_cross_compiler=yes
 
 # Find the build and target subdir names.
 
@@ -7667,7 +7662,7 @@
 # being built; programs in there won't even run.
 if test "${build}" = "${host}" && test -d ${srcdir}/gcc; then
   # Search for pre-installed headers if nothing else fits.
-  FLAGS_FOR_TARGET=$FLAGS_FOR_TARGET' -B$(build_tooldir)/bin/ -B$(build_tooldir)/lib/ -isystem $(build_tooldir)/include -isystem $(build_tooldir)/sys-include'
+  FLAGS_FOR_TARGET=$FLAGS_FOR_TARGET' -B$(build_tooldir)/bin/ -isystem $(build_tooldir)/include -isystem $(build_tooldir)/sys-include'
 fi
 
 if test "x${use_gnu_ld}" = x &&
